test/DFPerfScenarios/Tests/ManySequencesTest.cs

```diff
@@ -1,80 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'm just seeing this whole project deleted, not moved. Am I missing something?

Reply:
This was a mistake. I thought the directory had gone stale since I was pulling the C# perf project from a different source.
However, after speaking with Varshi, I learned this directory had not gone stale but was simply missing some of the scenarios.

In any case, I undid the removal just now!